README.md

```diff
@@ -70,7 +70,31 @@ Plugin 'iamcco/markdown-preview.nvim'
 :PluginInstall
 :call mkdp#util#install()
 ```
+Or with [lazy.nvim](https://github.com/folke/lazy.nvim):
 
+Add this in your `init.lua or plugins.lua`
+
+```lua
+-- install without yarn or npm
+{
+    "iamcco/markdown-preview.nvim",
+    cmd = { "MarkdownPreviewToggle" },
+    ft = { "markdown" }
```

Comment:
just need to add a comma here.

Reply:
oh, thanx, I've just added the comma you menthioned.